src/Umbraco.Core/Constants-Icons.cs

```diff
@@ -59,6 +59,26 @@ public static class Icons
             /// </summary>
             public const string MediaFile = "icon-document";
 
+            /// <summary>
+            /// System media video icon
+            /// </summary>
+            public const string MediaVideo = "icon-video";
+
+            /// <summary>
+            /// System media sound icon
+            /// </summary>
+            public const string MediaSound = "icon-sound-waves";
+
+            /// <summary>
+            /// System media document icon
+            /// </summary>
+            public const string MediaDocument = "icon-article";
+
+            /// <summary>
+            /// System media vector icon
+            /// </summary>
+            public const string MediaVector = "icon-picture";
```

Comment:
It would be great if this could use a different icon since it is a different media type, maybe a shape icon as SVG could be any graphic, icon, etc. and not neccesary a photo/picture.

Reply:
Could this Geometric icon be used?
![image](https://user-images.githubusercontent.com/6791648/112969757-d7904e00-914d-11eb-9cac-91282445030f.png)

Reply:
Otherwise this could be one:
![image](https://user-images.githubusercontent.com/6791648/112969906-fabafd80-914d-11eb-9073-c8fa71f3ee84.png)

Reply:
Yeah, one of those could work. It is difficult to pick the right icon, but at least it would be great with a different icon for each media type.

I noticed Nik asked on Snack for a specific media type for archives.
Should we have one for zip, rar etc. which could use `icon-compress`.

Reply:
Some other archive file formats are `.tar`, `.iso` or `.7z`.